app/components/markdown/markdown_table/markdown_table.js

```diff
@@ -122,13 +123,28 @@ export default class MarkdownTable extends React.PureComponent {
                         changeOpacity(this.props.theme.centerChannelColor, 0.0),
                         changeOpacity(this.props.theme.centerChannelColor, 0.1),
                     ]}
-                    style={style.moreBelow}
+                    style={[style.moreBelow, {width: this.getTableWidth()}]}
                 />
             );
         }
 
+        const expandButton = (
+            <TouchableOpacity
+                onPress={this.handlePress}
+                style={{...style.expandButton, left: this.state.containerWidth - 20}}
+            >
+                <Icon
+                    name={'expand'}
+                    style={style.icon}
+                />
+            </TouchableOpacity>
+        );
+
         return (
-            <TouchableOpacity onPress={this.handlePress}>
+            <TouchableOpacity
+                style={{paddingRight: 10}}
```

Comment:
Can you move this into the StyleSheet?

Reply:
Ah yes I definitely should have done that to begin with, thanks for pointing it out!